src/openseadragon.js

```diff
@@ -2269,29 +2271,6 @@ function OpenSeadragon( options ){
             event.stopPropagation();
         },
 
-        // Deprecated
```

Comment:
I started removal of some deprecated things that seem to be there already for a long time, it seems like now is a good time for it. I realized this will be better when the webgl PR is merged, so the merge is easier, then I will try to go through deprecated things and remove them if they are in OSD for many years.

Reply:
Sounds great!